src/Components/Job/Job.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { withRouter } from "react-router-dom";
+import styled from "styled-components";
+import { USER_FILTER_API } from "../../config";
+
+const Job = ({ job, setJobLoading, history }) => {
+  const {
+    id,
+    title,
+    name,
+    city,
+    likes_count,
+    image_url,
+    compensation,
+    like_status,
+    response_rate
+  } = job;
+
+  const onLike = () => {
+    fetch(USER_FILTER_API, {
+      method: like_status ? "DELETE" : "POST",
+      body: JSON.stringify({
+        company_id: id
+      }),
+      headers: {
+        Authorization: localStorage.getItem("token")
+      }
+    })
+      .then(response => response.json())
+      .then(result => {
+        setJobLoading(true);
+      });
```

Comment:
result를 쓰지 않으니까 result 없애주셔도 되겠네요! 
```suggestion
      .then(()=> setJobLoading(true));
```

Reply:
result에 success messege 받으면 실행하도록 수정했습니다!